src/app/static/src/app/components/modelOutput/ModelOutput.vue

```diff
@@ -17,8 +17,8 @@
                 <choropleth></choropleth>
             </div>
 
-            <div class="col" v-if="selectedTab==='Bar'">
-                Bar chart coming soon!
+            <div id="barchart-container" :class="selectedTab==='Bar' ? 'col-md-12' : 'd-none'">
```

Comment:
So this was because we don't want the filters to reset when the user tabs away right? Seems fine as a first pass, we might want to think about a way of passing selected filter options as props instead so we can persist them.

Reply:
Yeah, we talked about doing something like that. I'll add a ticket to persist barchart filters.